lib/fastlane/plugin/synx/actions/synx_action.rb

```diff
@@ -12,7 +12,10 @@ def self.run(params)
         cmd << ["--no-default-exclusions"] if params[:no_default_exclusions]
         cmd << ["--no-sort-by-name"] if params[:no_sort_by_name]
         cmd << ["--quiet"] if params[:quiet]
-        cmd << ["--exclusion #{params[:exclusion]}"] if params[:exclusion]
+        if params[:exclusion]
+          Array(params[:exclusion]).each {|exclusion|
+            cmd << ["--exclusion #{exclusion}"] }
```

Comment:
@tero2222 would your solution work for pathnames with spaces in it? don't you think it would be better if you enclose the exclusion in quotes? So that the command would look like this:
`$ synx -p --exclusion "/OCMock/Core Mocks" Source/OCMock.xcodeproj/`